### PR TITLE
New article: managing contacts

### LINKED ISSUE #162: Article about deleting contacts
Came across a support request inquiring about being unable to delete contacts from their account, which led to discovering useful information such as the fact that you cannot delete a contact if it's the current information for the registrant.

### PR DESCRIPTION
**Pull request by sbastn**
This PR introduces a new article that explains how to manage a contact. It explains the importance of a having correct contact info and well as how to create, edit, and delete a contact.

Closes #162

### CODE REVIEW

**Comment by aeden** [comment]
@sbastn what do you think of renaming the file to `contact-management.markdown` so it is grouped together with other contact-related articles?

**Comment by aeden** [comment]
Also, what do you think about adding a "Contacts" item to the left-hand menu and including this and the other contact articles in that section?

**Comment by sbastn** [comment]
Thanks for the feedback @aeden! I have added a new category called "Contacts" and added a couple of articles that I think are contact related. I didn't want to pull in the WHOIS articles since we already have a category for them.

screen shot 2017-10-03 at 1 41 18 pm

File: content/articles/contact-management.markdown [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+![creating a new contact](/files/change-contact-1.jpg)
+
+## Updating contacts
+
+If a contact is associated with a domain, updating that contact updates all the contact information associated with these domains. You don't need to individually update these domains if they point to the same contact.
+
+If you wish to update a specific domain only, you can either [replace the contact entirely](/article/changing-domain-contact/#replacing-a-domain-contact) with a new one, or [update the contact](/articles/changing-domain-contact/#updating-a-domain-contact) like we just mentioned to make that change.
+
+### Contact lock
```
Comment (weppos) [inline content/articles/contact-management.markdown]: I would change this title to:

    Domain registrant change and ICANN transfer lock

**Pull request merged**